can-bus/json/PDM/PDM_tx.json

```diff
@@ -12,17 +12,34 @@
             }
         }
     },
-    "PDM_Errors": {
-        "msg_id": 403, 
-        "cycle_time": 1000,
-        "description": "Errors on the PDM.",
+    "PDM_CanFifoOverflow": {
+        "msg_id": 403,
+        "cycle_time": 5000,
```

Comment:
cycle time of 5000 or 1000 for this message? I'll propagate the change to the BMS as well